File: 01 SERVIDOR/API-COMERCIALIZADORA/Models/Electrodomestico.cs
using System.ComponentModel.DataAnnotations;
using System.Runtime.Serialization;

namespace API_Comercializadora.Models;

[DataContract]
public class Electrodomestico
{
    [DataMember]
    [Key]
    public int Id { get; set; }

    [DataMember]
    [Required]
    [MaxLength(100)]
    public string Nombre { get; set; } = string.Empty;

    [DataMember]
    [MaxLength(250)]
    public string? Descripcion { get; set; }

    [DataMember]
    [MaxLength(100)]
    public string? Marca { get; set; }

    [DataMember]
    [Range(0, double.MaxValue)]
    public decimal PrecioVenta { get; set; }

    [DataMember]
    public bool Activo { get; set; } = true;

    [DataMember]
    public DateTime FechaRegistro { get; set; } = DateTime.UtcNow;
}

File: SOAP_DOTNET/03 CLIMOV-comer/app/src/main/java/ec/edu/espe/comercializadora/models/Electrodomestico.kt
package ec.edu.espe.comercializadora.models

data class Electrodomestico(
    val id: Int = 0,
    val nombre: String = "",
    val descripcion: String? = null,
    val marca: String? = null,
    val precioVenta: Double = 0.0,
    val activo: Boolean = true,
    val fechaRegistro: String = ""
)
